### PR TITLE
red(is) bull - added redis to backend, added bull to backend

### PR DESCRIPTION
**Pull request by steven4354**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,21 +6,26 @@
...
     "@hapi/hapi": "^18.4.0",
     "@types/rc": "^1.1.0",
     "blipp": "^4.0.1",
+    "bull": "^3.12.1",
     "faker": "^4.1.0",
     "knex": "^0.20.6",
     "lodash": "^4.17.15",
     "mongoose": "^5.8.4",
     "mysql2": "^2.1.0",
+    "nodemon": "^2.0.2",
```
Comment (zachary95) [inline package.json]: This should be a devDependancy

File: src/services/GarageService.ts [hunk 2]
```diff
@@ -17,4 +20,24 @@ export default class GarageService {
...
+  // use the jobs
+  async repairCar(car) {
+    const myJob = carRepairQueue.add({ car });
+    console.log("STEVENDEBUG myJob ", myJob);
+
+    return myJob;
+  }
+
+  async checkTirePressureEvery3Months(car) {
+    const myJob = carTirePressureQueue.add({ car }, {
```
Comment (zachary95) [inline src/services/GarageService.ts]: Can this be added into the carTirePressureQueue? So it's global to all queue pushes

Comment (steven4354) [reply src/services/GarageService.ts]: @zachary95 do you mean to make the repeat inherent in the carTirePressureQueue?

Comment (steven4354) [reply src/services/GarageService.ts]: I think the repeat is set on the .add

Comment (zachary95) [reply src/services/GarageService.ts]: Yes – moving the schedule instructions in the job file rather than where you add to the queue. Is it possible?

**Pull request merged**